### PR TITLE
docs: Improve @rest-hooks/hooks README

### PR DESCRIPTION
**Pull request by ntucker**
- Fix title
- Add explanations

### CODE REVIEW

File: packages/hooks/README.md [hunk 1]
```diff
@@ -16,19 +16,27 @@ Composable hooks for networking data
 
 ### useCancelling()
 
+[Aborts](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) inflight request is the parameters
+change.
```
Comment (ljharb) [inline packages/hooks/README.md:20]: nitL: hard wrapping seems weird and increases diff churn
```suggestion
[Aborts](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) inflight request is the parameters change.
```

**Review by ljharb — APPROVED** [review]

File: packages/hooks/README.md (hunk 1 above)
Comment (ntucker) [reply packages/hooks/README.md:20]: What is hard wrapping?

Comment (ljharb) [reply packages/hooks/README.md]: adding a newline in the middle of a sentence/paragraph - usually to ensure line lengths are under an arbitrary threshold. Instead, it's best if a sentence/paragraph is as long as it is, allowing the display side to do the line breaking.

Comment (ntucker) [reply packages/hooks/README.md]: Oh I should probably change my editor settings for .md files, I usually randomly put endlines cuz it doesn't wordwrap.

**Pull request merged**